imports/ui/components/organisms/LogoBox.jsx

```diff
@@ -70,6 +125,12 @@ export const LogoBox = ({ logo }) =>
           : 'top-center'
       }
     >
-      <Logo image={logo.image} size={logo.size} />
```

Comment:
Same here. In the original application, the image is master and will be shown if exists only otherwise the text logo is visible.